### ISSUE TITLE
Add yarn to the nix-shell

### ISSUE THREAD

**Issue by rossabaker** [http4s/http4s#5235]
I can't build locally:

```
[error] (ember-serverJS / Test / npmInstallDependencies) java.io.IOException: Cannot run program "yarn" (in directory "/Users/ross.baker/src/http4s/ember-server/js/target/scala-2.12/scalajs-bundler/test"): error=2, No such file or directory
```

This probably falls on me, but we also should have an environment setup for people who don't want one out of the box.

**Comment by armanbilge** [comment]
Yarn is needed for installing a websockets client to test the websockets server. We may be able to remove this after https://github.com/http4s/http4s/issues/5227.

**Comment by rossabaker** [comment]
Fixed by #5238
